db/seeds/fibonacci/exercise_spec_3.rb

```diff
@@ -4,7 +4,7 @@
 require './reference'
 
 describe '#fibonacci' do
-  let(:sample_count) { 32 }
+  sample_count = 32
```

Comment:
Thanks! What about using a constant here (and moving it outside the `describe` block)?

```suggestion
  SAMPLE_COUNT = 32
```